### PR TITLE
NTP sync test rebuilt

### PR DESCRIPTION
**Pull request by vkhalin**
rebuilt ntp checks to work with pytest, removed useless code. Need to be tested on the env to be sure that it works fine

### CODE REVIEW

**Comment by vkhalin** [comment]
Few commits to me merged. MTU and NTP scripts are rewritten and need to be tested

**Comment by vkhalin** [comment]
@msenin94 please review

**Comment by vkhalin** [comment]
@msenin94 please review changes n merge. Thanks!

File: mk_verificator/tests/networking/mtu/test_mtu.py [hunk 1]
```diff
@@ -0,0 +1,177 @@
+#!/usr/bin/env python
+
+import salt.client as client
+import texttable as tt
+
+expectations = {
```
Comment (msenin94) [inline mk_verificator/tests/networking/mtu/test_mtu.py]: let's merge utils first. after that I suggest to separate logic and move this large json to the divided file

File: mk_verificator/tests/networking/mtu/test_mtu.py [hunk 2]
```diff
@@ -0,0 +1,177 @@
...
+    },
+    "mon": {
+        "eth0": "1500",
+        "eth1": "9000"
+    }
+}
+
+
+def test_mtu(local_salt_client):
+    skipped_ifaces = ["bonding_masters", "lo"]
```
Comment (msenin94) [inline mk_verificator/tests/networking/mtu/test_mtu.py]: let's separate this test per node-group as "minimal" monatomic action. then let's use parametrize decorator  http://doc.pytest.org/en/latest/parametrize.html

File: mk_verificator/tests/system/test_pkg_version.py [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+    groups = {}
+
+    missed_pkgs, version_conflicts = [], []
+
+    for node_name, node_pkgs in pkgs_info.items():
+        group_name = node_name.split('-')[0]
+        if not groups.has_key(group_name):
+            groups[group_name] = [(node_name, node_pkgs)]
+        else:
+            groups[group_name].append((node_name, node_pkgs))
```
Comment (dkulykovmirantis) [inline mk_verificator/tests/system/test_pkg_version.py]: This part is unnecessary
utils.get_groups returns groups.
For example: [cpu*, ctl*, ...]

File: mk_verificator/tests/system/test_pkg_version.py [hunk 4]
```diff
@@ -0,0 +1,52 @@
...
+
+    for group_name, nodes in groups.items():
+        if len(nodes) > 1:
+            for node_i in nodes:
+                for node_j in nodes:
+                    if node_i[0] == node_j[0]:
+                        continue
+
+                    node_i_name, node_j_name = node_i[0], node_j[0]
```
Comment (dkulykovmirantis) [inline mk_verificator/tests/system/test_pkg_version.py]: need discussion

File: mk_verificator/tests/networking/mtu/test_mtu.py [hunk 5]
```diff
@@ -0,0 +1,54 @@
...
+    TOTAL = {}
+    failed_ifaces = {}
+
+    gauges = utils.get_expected_mtu(__file__)
+
+    network_info = local_salt_client.cmd(groups, 'cmd.run', ['sudo ls /sys/class/net/'])
+
+    for node, ifaces_info in network_info.iteritems():
+        if 'kvm' in node:
+            kvm_info = local_salt_client.cmd(node, 'cmd.run',
```
Comment (msenin94) [inline mk_verificator/tests/networking/mtu/test_mtu.py]: please, use pep8 for verification

File: mk_verificator/tests/networking/mtu/test_mtu.py [hunk 6]
```diff
@@ -0,0 +1,54 @@
...
+                                             ["virsh list | grep jse2 | awk '{print $2}' | xargs -n1 virsh domiflist | "
+                                              "grep -v br-pxe | grep br- | awk '{print $1}'"])
+            ifaces_info = kvm_info.get(node)
+        node_name = node.split('-')[0]
+        node_ifaces = ifaces_info.split('\n')
+        if node_name not in gauges:
+            print "Node {} is not matching expected groups!".format(node)
```
Comment (msenin94) [inline mk_verificator/tests/networking/mtu/test_mtu.py]: please, do not use print in the tests

File: mk_verificator/tests/system/test_pkg_version.py [hunk 7]
```diff
@@ -0,0 +1,52 @@
...
+
+                    if missed_pkgs:
+                        assert not missed_pkgs, "Pkgs mismatch for node {} and {}:\n{}"\
+                            .format(node_i_name, node_j_name, missed_pkgs)
+
+                    if version_conflicts:
+                        assert not version_conflicts, "Version conflicts for node {} and {}:\n{}"\
+                            .format(node_i_name, node_j_name, version_conflicts)
+        else:
+            print "Verification for group %s was skipped due to count of nodes less than 2" % group_name
```
Comment (msenin94) [inline mk_verificator/tests/system/test_pkg_version.py]: please, do not use print

File: mk_verificator/tests/networking/mtu/test_mtu.py [hunk 8]
```diff
@@ -0,0 +1,54 @@
+import pytest
+from mk_verificator import utils
+
+
+@pytest.mark.parametrize(
+    "groups",
+    utils.get_groups(utils.get_configuration(__file__))
+)
+def test_mtu(local_salt_client, groups):
```
Comment (msenin94) [inline mk_verificator/tests/networking/mtu/test_mtu.py]: that's group, not "groups"

Comment (msenin94) [inline mk_verificator/tests/networking/mtu/test_mtu.py]: not groups. use singular form

File: mk_verificator/tests/networking/mtu/test_mtu.py [hunk 9]
```diff
@@ -0,0 +1,54 @@
...
+    skipped_ifaces = ["bonding_masters", "lo"]
```
Comment (msenin94) [inline mk_verificator/tests/networking/mtu/test_mtu.py]: please move to the config and use global config like
*import group*
test_config = utils.get_configuration(__file__)

def test_1():
    skipped_ifaces = config['skipped_ifaces']

File: mk_verificator/tests/networking/mtu/test_mtu.py [hunk 10]
```diff
@@ -0,0 +1,54 @@
...
+    TOTAL = {}
+    failed_ifaces = {}
+
+    gauges = utils.get_expected_mtu(__file__)
```
Comment (msenin94) [inline mk_verificator/tests/networking/mtu/test_mtu.py]: let's use get_expected_results or something like this

**Pull request closed without merging**